test/Microsoft.ML.Benchmarks/TrainPredictionBench.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Running;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public class StochasticDualCoordinateAscentClassifierBench
+    {
+        internal static ClassificationMetrics s_metrics;
+        private static PredictionModel<IrisData, IrisPrediction> s_trainedModel;
+        private static string s_dataPath;
+
+        [Benchmark]
+        public void PredictIris()
+        {
+            IrisPrediction prediction = s_trainedModel.Predict(new IrisData()
+            {
+                SepalLength = 3.3f,
+                SepalWidth = 1.6f,
+                PetalLength = 0.2f,
+                PetalWidth = 5.1f,
+            });
```

Comment:
nit: Probably better to create those separately as static enumeration of test data.  that way you don't measure the object creation time
Then you can easily call different methods on differnt length of prediction enums or single objects by calling TestData.Take(5);

Reply:
sounds good

---
In reply to: [188046953](https://github.com/dotnet/machinelearning/pull/62#discussion_r188046953) [](ancestors = 188046953)